ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```
@@ -285,15 +285,20 @@ http_proxy=${proxy} https_proxy="${proxy}" HTTP_PROXY="${proxy}" HTTPS_PROXY="${
   oinst create cluster &
 
 if ! wait $!; then
-  exit 1
+  echo "\n[INFO] Waiting for install-complete timed out. Install is still running....."
```

Shellcheck (0.11.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)